zstacklib/zstacklib/utils/pci.py

```
@@ -38,9 +40,256 @@ def update_cache_devices(devices_dict):
+    """Check if npu-smi output contains the PCI address"""
+    try:
+        r, o, _ = bash_roe("npu-smi info -l")
+        if r == 0 and o:
+            # Normalize PCI address for comparison
+            normalized_pci = pci_address.lower()
+            # Check if PCI address is in output (handle various formats like "PCIe Bus Info: 0000:81:00.0")
+            output_lower = o.lower()
+            if normalized_pci in output_lower:
+                return True
+    except Exception:
+        pass
```

Ruff (0.14.13)

`try`-`except`-`pass` detected, consider logging the exception

(S110)